src/tawny/bubo/xls.clj

```diff
@@ -0,0 +1,21 @@
+(ns tawny.bubo.xls
+  (:require
+   [clojure.string :as str]
+   [clojure.java.io :as io]))
+(use 'dk.ative.docjure.spreadsheet)
+
+(defn- read-workbook [filename sheet]
+  (->> (load-workbook filename)
+       (select-sheet sheet)
+       row-seq
+       (map cell-seq)
+       (map #(map read-cell %))))
+
+;(def workbook (load-workbook "xls-test.xlsx"))
+;(def sheet (select-sheet "Sheet1" workbook))
+
+(defn xls-apply [f filename sheet]
```

Comment:
This needs to use keyword arguments, as we will need to add further optional arguments.

Reply:
This is updated in the new version.